include/unifex/just.hpp

```diff
@@ -72,26 +72,38 @@ class _sender<Values...>::type {
   template <template <typename...> class Variant>
   using error_types = Variant<std::exception_ptr>;
 
-  template <typename... Values2>
-  explicit type(Values2&&... values)
+  type() = default;
+  type(const type&) = default;
+  type(type&&) = default;
+
+  template <
+    typename... Values2,
+    std::enable_if_t<
+      (sizeof...(Values2) == sizeof...(Values) &&
+      sizeof...(Values) > 1), int> = 0>
+    explicit type(Values2&&... values)
     noexcept(std::is_nothrow_constructible_v<std::tuple<Values...>, Values2...>)
     : values_((Values2 &&) values...) {}
 
-  template <typename Receiver>
+  template <
+    typename Value2,
+    std::enable_if_t<
+      (sizeof...(Values) == 1 &&
+      !std::is_same_v<Value2, type>), int> = 0>
+    explicit type(Value2&& value)
+    noexcept(std::is_nothrow_constructible_v<std::tuple<Values...>, Value2>)
+    : values_((Value2 &&) value) {}
```

Comment:
I wonder whether this could all be simplified by adding an extra std::in_place_t arg in front of the parameter pack to disambiguate the in-place constructors from the copy/move constructors?

e.g.
```
template<typename... Values2>
explicit type(std::in_place_t, Values2&&... values)
  noexcept(std::is_nothrow_constructible_v<std::tuple<Values...>, Values2...>)
  : values_((Values2&&)values...)
{}
```

Then just update the `just()` implementation to pass `std::in_place` as the first arg to the constructor.